package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "hdclient",
+  "engines": {
+    "node": ">=6.9.5"
+  },
+  "version": "0.1.0",
+  "description": "Hyperdrive appliance client library",
+  "main": "index.js",
+  "scripts": {
+    "lint": "eslint $(git ls-files '*.js')",
+    "test": "mocha --use_strict --check-leaks --no-exit "
+  },
+  "repository": "scality/hdclient",
+  "author": "Maxime Lubin",
+  "dependencies": {
+    "werelogs": "scality/werelogs"
```

Comment:
Was a model chosen for the dependency handling of hdclient ? Are you going to depend on a fixed version of Werelogs, or are you going to do the same (mistake?) as what was done in our S3C components, depending on moving branches ?

Reply:
Frankly I have no idea. I adapted the package.json from Sproxyd... But yeah, seems like a good idea to fix the version.